packages/extension-sdk/src/connect/extension_host_api.ts

```diff
@@ -52,54 +60,89 @@ export class ExtensionHostApiImpl implements ExtensionHostApi {
   private chattyHost: ChattyHostConnection
   private setInitialRoute?: (route: string, routeState?: any) => void
   private hostChangedRoute?: (route: string, routeState?: any) => void
+  private visualizationDataReceivedCallback?: VisualizationDataReceivedCallback
+  private _visualizationSDK?: VisualizationSDK
+  private _tileSDK?: TileSDK
+
   private contextData?: string
 
   constructor(configuration: ExtensionHostApiConfiguration) {
     this._configuration = configuration
-    const { chattyHost, setInitialRoute, hostChangedRoute } =
-      this._configuration
+    const {
+      chattyHost,
+      setInitialRoute,
+      hostChangedRoute,
+      visualizationDataReceivedCallback,
+    } = this._configuration
     this.chattyHost = chattyHost
     this.setInitialRoute = setInitialRoute
     this.hostChangedRoute = hostChangedRoute
+    this.visualizationDataReceivedCallback = visualizationDataReceivedCallback
+  }
+
+  get visualizationSDK(): VisualizationSDK | undefined {
```

Comment:
when would this return undefined?

Reply:
It would not. A left over from the original implementation when I only created it if visualizationDataReceivedCallback was set. Will fix